src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,37 @@
 package core.basesyntax.service;
 
-import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
```

Comment:
I don't see a test class with tests
<img width="633" alt="Screenshot 2022-06-20 at 20 45 46" src="https://user-images.githubusercontent.com/60258373/174655171-bb673c66-c67a-49d4-92e2-943cc879a160.png">

Reply:
Sorry, I forgot to commit test class